engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -16,9 +16,12 @@ <h4 translate>Basic Information</h4>
             on-save="save(host)">
         </dd>
 
-        <dt translate>UUID</dt>
+        <dt translate>Subscription UUID</dt>
         <dd>{{ host.subscription_facet_attributes.uuid }}</dd>
 
+        <dt translate>System UUID</dt>
```

Comment:
I might suggest 'Hardware UUID' or something to that affect?  "Bios UUID"? "DMI UUID"?

Reply:
@jlsherrill 'Hardware UUID' may raise questions in case of virtual machines, So changed it to "Bios UUID".

Reply:
Sounds good!